Description: use newer release of node module cheerio
Forwarded: no
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2021-02-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/lib/formatters/html.js
+++ b/tests/lib/formatters/html.js
@@ -525,7 +525,7 @@
             const result = formatter(code.results, { rulesMeta });
             const $ = cheerio.load(result);
 
-            checkContentRow($, $("tr")[1], { group: "f-0", lineCol: "5:10", color: "clr-2", message: "Unexpected &lt;&amp;&quot;&apos;&gt; foo.", ruleId: "foo" });
+            checkContentRow($, $("tr")[1], { group: "f-0", lineCol: "5:10", color: "clr-2", message: "Unexpected &lt;&amp;\"'&gt; foo.", ruleId: "foo" });
         });
     });
 
